snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedReceivedSnapshot.java

```diff
@@ -169,12 +172,26 @@ private boolean isSnapshotIdInvalid(final String snapshotId) {
   }
 
   private boolean writeReceivedSnapshotChunk(
-      final SnapshotChunk snapshotChunk, final Path snapshotFile) throws IOException {
-    Files.write(
-        snapshotFile,
-        snapshotChunk.getContent(),
-        StandardOpenOption.CREATE_NEW,
-        StandardOpenOption.WRITE);
+      final SnapshotChunk snapshotChunk, final Path snapshotFile) {
+    try (var channel =
+        FileChannel.open(
+            snapshotFile,
+            StandardOpenOption.CREATE_NEW,
+            StandardOpenOption.WRITE,
+            StandardOpenOption.SYNC)) {
```

Comment:
If we open it in SYNC mode, will it basically sync after every `write` operation? Can we avoid this and sync only once at the end when we've written the whole file?